src/main/resources/static/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>WATCHTUBE</title>
+    <meta charset="UTF-8">
+    <link href="style.css" type="text/css" rel="stylesheet"/>
+	
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+
+</head>
+<body>
+
+<div id="header"></div>
+
+<div id="signUpForm" class="forms"></div>
+
+<div id="userSearchForm" class="forms"></div>
+
+<div id="videosContent"></div>
+
+<script src="renderOntoDivs.js" type="text/babel"></script>
```

Comment:
Sorry forgot to rename it. It should be "initial render.js". I am used to that when i refactor class name in IDEA, it rename all usages in project:)